src/style.css

```
@@ -1 +1,34 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.start__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > :nth-child(1),
+.stars--2 > :nth-child(5n-4),
+.stars--2 > :nth-child(5n-3),
+.stars--3 > :nth-child(5n-4),
+.stars--3 > :nth-child(5n-3),
+.stars--3 > :nth-child(5n-2),
```

```suggestion
.stars--3 > :nth-child(-n + 3),
```
you can just simplify it